Felienne/hedy
# [BUG FIX] Always shows newest users first in admin users overview

**Description**
In this PR we make a bucketfix for the issue where the newest users where not automatically shown when loading all users on the admin page. This is due to Dynamo not being sorted automatically on the `created` attribute. The server load increases a bit in this case as previously we only retrieved the first 500 users automatically. Now we always retrieve all users and sort ourselves.

**Fixes**
This PR fixes #2641.

**How to test**
Navigate to the users admin page and verify that the newest users are always shown on top.
